apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -12540,8 +12537,8 @@
 <e>       <p><l>koʻrinish<s n="n"/></l>                <r>kórinis<s n="n"/></r></p></e>
 <e>       <p><l>koʻrinishli<s n="adj"/></l>            <r>kórinisli<s n="adj"/></r></p></e>
 <e>       <p><l>koʻrish<s n="n"/><s n="vn"/></l>       <r>kóriw<s n="n"/><s n="vn"/></r></p></e>
-<e>       <p><l>koʻrishish<s n="v"/><s n="iv"/><s n="coop"/></l><r>kóriwiw<s n="v"/><s n="iv"/><s n="coop"/></r></p></e>
-<e>       <p><l>koʻrishtir<s n="v"/><s n="tv"/><s n="caus"/></l><r>kóriwtir<s n="v"/><s n="tv"/><s n="caus"/></r></p></e>
+<e>       <p><l>koʻrishish<s n="v"/><s n="iv"/><s n="coop"/></l><r>kórisiw<s n="v"/><s n="iv"/><s n="coop"/></r></p></e>
+<e>       <p><l>koʻrishtir<s n="v"/><s n="tv"/><s n="caus"/></l><r>kóristir<s n="v"/><s n="tv"/><s n="caus"/></r></p></e>
```

Comment:
These still look wrong.  See my comment from before.  Specifically:
> I think the correct solution is to do it like this:
> 
> ```
> <p><l>koʻr<s n="v"/><s n="iv"/><s n="coop"/></l><r>kór<s n="v"/><s n="iv"/><s n="coop"/></r></p>
> ```
> 
> But it depends what's in the monolingual dictionaries. If they both have productive cooperative forms (meaning that e.g. kórisiw is analysed as kór), then this is fine, but also perhaps superfluous, since it's handled by the bidix entry for kór. If the cooperative forms are analysed as separate verbs, then you'll need an entry for kóris.